evalml/tuners/random_search_tuner.py

```diff
@@ -19,7 +19,7 @@ def __init__(self, pipeline_hyperparameter_ranges, random_state=0, with_replacem
 
         Arguments:
             pipeline_hyperparameter_ranges (dict): a set of hyperparameter ranges corresponding to a pipeline's parameters
-            random_state: Unused in this class
+            random_state (int): Unused in this class. Defaults to 0.
             with_replacement: If false, only unique hyperparameters will be shown
             replacement_max_attempts: The maximum number of tries to get a unique
                 set of random parameters. Only used if tuner is initalized with
```

Comment:
I know it's out of scope, but could you do me a huge favor and add some type hinting to these while you're here? Save me from filing an issue, lol.